BREAKING.md

```diff
@@ -1,12 +1,17 @@
 ## 0.28 Breaking Changes
 
+- [ODSP Driver IPersistedCache changes](#ODSP-Driver-IPersistedCache-Changes)
 - [IFluidPackage Changes](#IFluidPackage-Changes)
 - [DataObject changes](#DataObject-changes)
 - [RequestParser](#RequestParser)
 - [IFluidLodable.url is removed](#IFluidLodable.url-is-removed)
 - [Loader Constructor Changes](#Loader-Constructor-Changes)
 - [Moving DriverHeader and merge with CreateNewHeader](#moving-driverheader-and-merge-with-createnewheader)
 
+### ODSP Driver IPersistedCache-Changes
+Added api `removeAllEntriesForDocId` which allows removal of all entries for a given document id. Also the schema for entries stored inside odsp `IPersistedCache` has changed.
+It now stores/expect values as `IPersistedCacheValueWithEpoch`. So host needs to clear its cached entries in this version.
```

Comment:
@christiango - FYI there is change in structure of data stored in cache, so old data would need to be purged on consuming this change (I assume 0.28)

Reply:
Can we track this work as part of what needs to happen with the 0.28 bump? How do we make sure this doesn't get lost?

Would be good to see if going forward things like this could be more automatic, such as making things like this be a cache miss or something like that.

Reply:
Actually, I'd like to challenge this a bit further here. Can we structure this change such that if FFX "forgets" to do the cache clearing, we don't fail catastrophically? Or is that already what happens here and you are just asking us to make sure old data gets cleared?

We should be clearing unused data already after some time has passed.

Reply:
The cache needs to be purged. Isn't it enough to add it in breaking.md on our side? We may track this by opening an issue but that will be in FFX repo?

Reply:
I think we would fail if the data is not cleared.

Reply:
So whoever is doing the bump, shouldn't he/she goes through breaking.md changes for the given version?

Reply:
We can add safety net / sort of version (v = 1) to payload and start ignoring  other versions.
That's actually good, as user might run v = 0, v = 1 & v = 2 of code on same box.
So if it's not too hard, I'd implement safe ignore of data in our layer

Reply:
Yes for protection, we can add the safety check and ignore on other version.

Reply:
Yeah, we should do the work to clear old data (it may happen naturally), but the more important thing to have here is a mechanism where "old" data isn't a cache hit. Would love a safe failure here and let our natural "cleanup" code get rid of the old data.

Reply:
Added version info and conversion to proper type for old cache entries.